python/test/unit/language/test_core_amd.py

```diff
@@ -1588,10 +1588,11 @@ def get_variant_golden(a, b):
     return c_padded[:SIZE_M, :SIZE_N]
 
 
-@pytest.mark.parametrize('SIZE_M,SIZE_N,SIZE_K,NUM_WARPS,BLOCK_SIZE_M,BLOCK_SIZE_N,BLOCK_SIZE_K', [
-    [64, 32, 128, 4, 64, 32, 64],
+@pytest.mark.parametrize('SIZE_M,SIZE_N,SIZE_K,NUM_WARPS,BLOCK_SIZE_M,BLOCK_SIZE_N,BLOCK_SIZE_K,NUM_STAGES', [
+    [64, 32, 128, 4, 64, 32, 64, 0],
+    [64, 32, 128, 4, 64, 32, 64, 2]
 ])
-def test_gemm(SIZE_M, SIZE_N, SIZE_K, NUM_WARPS, BLOCK_SIZE_M, BLOCK_SIZE_N, BLOCK_SIZE_K):
+def test_gemm(SIZE_M, SIZE_N, SIZE_K, NUM_WARPS, BLOCK_SIZE_M, BLOCK_SIZE_N, BLOCK_SIZE_K, NUM_STAGES):
```

Comment:
@sjw36 
FYI: This test is named gemm, but in reality it is not complete and limited, it works on one CTA only.
Feel free to improve this test if you need it :)

I've used it to verify that pipeline pass(old one) produces working code and that MFMA dot tiles across K dimension correctly.